### ISSUE TITLE
Warning messages encountered while installing on RHEL 8.5

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#2885]
Warning messages encountered while installing on RHEL 8.5:

```
WARNING: The scripts pip, pip3 and pip3.10 are installed in '/usr/local/bin' which is not on PATH.
Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.

WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
```

Certainly removing local pip install is a big part of the answer, but we can't do that all at once.
